js/cryptocom.js

```diff
@@ -242,6 +243,12 @@ module.exports = class cryptocom extends Exchange {
                     'swap': 'DERIVATIVES',
                     'future': 'DERIVATIVES',
                 },
+                'networks': {
+                    'ERC20': 'ETH',
```

Comment:
this is only the case for `ETH` and that is already handled elsewhere so I think we can safely remove this from here

Reply:
Is this already handled? Where?
Cause when I run `node examples/js/cli cryptocom fetchDepositWithdrawFees '["ETH"]'`:
```js
Node.js: v14.20.1
CCXT v2.2.40
cryptocom.fetchDepositWithdrawFees (ETH)
2022-12-15T17:14:40.880Z iteration 0 passed in 1487 ms

{
  ETH: {
    info: {
      full_name: 'Ethereum',
      default_network: 'ETH',
      network_list: [
        {
          network_id: 'Arbitrum',
          withdrawal_fee: '0.00100000',
          withdraw_enabled: true,
          min_withdrawal_amount: '0.002',
          deposit_enabled: true,
          confirmation_required: '0'
        },
        {
          network_id: 'CRONOS',
          withdrawal_fee: '0.00010000',
          withdraw_enabled: true,
          min_withdrawal_amount: '2.0E-4',
          deposit_enabled: true,
          confirmation_required: '0'
        },
        {
          network_id: 'ETH',
          withdrawal_fee: '0.00400000',
          withdraw_enabled: true,
          min_withdrawal_amount: '0.008',
          deposit_enabled: true,
          confirmation_required: '0'
        },
        {
          network_id: 'MATIC',
          withdrawal_fee: '0.00010000',
          withdraw_enabled: true,
          min_withdrawal_amount: '2.0E-4',
          deposit_enabled: true,
          confirmation_required: '0'
        },
        {
          network_id: 'OP',
          withdrawal_fee: '0.00040000',
          withdraw_enabled: true,
          min_withdrawal_amount: '8.0E-4',
          deposit_enabled: true,
          confirmation_required: '0'
        }
      ]
    },
    withdraw: { fee: undefined, percentage: undefined },
    deposit: { fee: undefined, percentage: undefined },
    networks: {
      Arbitrum: {
        deposit: { fee: undefined, percentage: undefined },
        withdraw: { fee: 0.001, percentage: false }
      },
      CRONOS: {
        deposit: { fee: undefined, percentage: undefined },
        withdraw: { fee: 0.0001, percentage: false }
      },
      ERC20: {
        deposit: { fee: undefined, percentage: undefined },
        withdraw: { fee: 0.004, percentage: false }
      },
      MATIC: {
        deposit: { fee: undefined, percentage: undefined },
        withdraw: { fee: 0.0001, percentage: false }
      },
      OP: {
        deposit: { fee: undefined, percentage: undefined },
        withdraw: { fee: 0.0004, percentage: false }
      }
    }
  }
}
```

Network `ETH` becomes `ERC20` which isn't what we have agreed on for currencies like `ETH`, `TRX`, ..., right?